lib/modules/apostrophe-events-calendar-widgets/lib/api.js

```diff
@@ -0,0 +1,44 @@
+var moment = require('moment');
+var _ = require('lodash');
+
+module.exports = function(self, options) {
+  self.getEvents = async function(req) {
+    var month;
+    if (req.query && !req.body.month) {
+      month = moment().format('YYYY-MM');
+    } else {
+      month = req.body.month
+    }
+
+    var max = moment(month).add({ days: 15, months: 1 }).format('YYYY-MM-DD');
+    var min = moment(month).subtract(15, 'd').format('YYYY-MM-DD');
+    var query = { startDate: { $lte: max, $gte: min } }
```

Comment:
I would think it would be startDate: $gte min, and endDate: $lte max. Otherwise you could miss The Big Conference that's still going on at the last day in range.

Reply:
did this